### PR TITLE
Lottetreg/improve stack generation

### PR DESCRIPTION
**Pull request by lottetreg**
## BACKGROUND
We've had two vendors recently submit scripts with the typo "stable" in them instead of their app's repo name. We've also just had a vendor submit scripts that weren't executable. Hopefully this will prevent vendors from running into the same issues in the future.

## Changes
* Make all stack template scripts executable so that vendors who run the generate script command will automatically get executable scripts
* remove the word "stable" from the generate script instructions so that vendors know to use their own repo name instead

-----------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

**Review by taylorjhnguyen — DISMISSED** [review]
lgtm

**Comment by scott** [comment]
I noticed that `deploy.sh` does not appear to be made executable?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ To add your application to the [DigitalOcean Marketplace](https://marketplace.di
 1. Create a git branch that includes the name of your app (example: `$APP_NAME-first-pr`)
 1. Run the following commands to create your `deploy.sh`, `upgrade.sh`, `uninstall.sh` and `values.yml` files. They will be located in `stacks/$APP_NAME`:
 ```bash
-export HELM_REPO_NAME=stable
-export HELM_REPO_URL=https://charts.helm.sh/stable
+export HELM_REPO_NAME=$REPO_NAME
+export HELM_REPO_URL=https://charts.helm.sh/your-helm-repo
```
Comment (scott) [inline CONTRIBUTING.md:16]: Maybe we want this to not assume the repo is located at `https://charts.helm.sh/` as many vendors now keep their charts on github page.  perhaps `$REPO_URL` would be better?

**Comment by lottetreg** [comment]
> I noticed that `deploy.sh` does not appear to be made executable?

deploy.sh was already executable :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (lottetreg) [reply CONTRIBUTING.md:16]: Yes, good point!

**Review by scott — APPROVED** [review]
LGTM 👍

**Pull request merged**